modules/update-restricted-network-cli.adoc

```diff
@@ -0,0 +1,186 @@
+  repositoryDigestMirrors:
+  - mirrors:
+    - <local_registry_host_name>:<local_registry_host_port>/<repository_name>/release
+    source: quay.io/openshift-release-dev/ocp-release
+  - mirrors:
+    - <local_registry_host_name>:<local_registry_host_port>/<repository_name>/release
+    source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
+----
+<1> Specify the entire specification that was showin in the output of the
+`oc adm mirror release` command that mirrored the images for this release.
+.. Create the `ImageContentSourcePolicy` resource that you defined:
++
+----
+$ oc create -f test.yaml
+----
+.. Confirm that the new `ImageContentSoucePolicy` is in effect by reviewing the node status:
++
+----
+$ oc describe node | grep machineconfig
+                    machineconfiguration.openshift.io/currentConfig: rendered-master-9194b32791755030dcce887c66024113
+                    machineconfiguration.openshift.io/desiredConfig: rendered-master-9194b32791755030dcce887c66024113
+                    machineconfiguration.openshift.io/reason:
+                    machineconfiguration.openshift.io/state: Done
+                    machineconfiguration.openshift.io/currentConfig: rendered-worker-a0f873210ec70fb117835a9c9527db29
+                    machineconfiguration.openshift.io/desiredConfig: rendered-worker-a0f873210ec70fb117835a9c9527db29
+                    machineconfiguration.openshift.io/state: Done
+                    machineconfiguration.openshift.io/currentConfig: rendered-master-9194b32791755030dcce887c66024113
+                    machineconfiguration.openshift.io/desiredConfig: rendered-master-9194b32791755030dcce887c66024113
+                    machineconfiguration.openshift.io/reason:
+                    machineconfiguration.openshift.io/state: Done
+                    machineconfiguration.openshift.io/currentConfig: rendered-worker-a0f873210ec70fb117835a9c9527db29
+                    machineconfiguration.openshift.io/desiredConfig: rendered-worker-a0f873210ec70fb117835a9c9527db29
+                    machineconfiguration.openshift.io/state: Done
+                    machineconfiguration.openshift.io/currentConfig: rendered-worker-a0f873210ec70fb117835a9c9527db29
+                    machineconfiguration.openshift.io/desiredConfig: rendered-worker-a0f873210ec70fb117835a9c9527db29
+                    machineconfiguration.openshift.io/state: Done
+                    machineconfiguration.openshift.io/currentConfig: rendered-master-9194b32791755030dcce887c66024113
+                    machineconfiguration.openshift.io/desiredConfig: rendered-master-9194b32791755030dcce887c66024113
+                    machineconfiguration.openshift.io/state: Done
+
+----
++
+For each machine, the `machineconfiguration.openshift.io/state` must be `Done` before you begin the update.
+
+. Start the cluster update:
++
+----
+$ oc adm upgrade --force --to-image <local_registry_host_name>:<local_registry_host_port>/ocp/release:<release_version> <1>
```

Comment:
For a product release, I think here no need --force option.

Reply:
it seems that for disconnected installations you still need `--force` at least with 4.2.0